MonitoringSystem/Client/MonitoringClient.py

```diff
@@ -0,0 +1,112 @@
+"""
+This is the client of the Monitoring service based on Elasticsearch.
+"""
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.DISET.RPCClient                 import RPCClient
+from DIRAC.Core.Utilities.Plotting.FileCoding   import codeRequestInFileId
+import types 
+
+class MonitoringClient( object ):
+  """ 
+  .. class:: MonitoringClient
+  
+  This class expose the methods of the Monitoring Service
+  
+  param: RPCClient __rpcClient stores the rpc client used to connect to the service
+  
+  """
+  
+  def __init__( self, rpcClient = None ):
+    self.__rpcClient = rpcClient
```

Comment:
You actually never set self.__rpcClient (unless given in the init), which is a good thing for round robin. But is it really a use case to pass rpcClient ?